### PR TITLE
SDIT-2123: 🔊 Reduce app insights logging

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: applicationinsights.dev.json [hunk 1]
```diff
@@ -16,16 +16,49 @@
...
+      {
+        "telemetryType": "dependency",
+        "attributes": [
+          {
+            "key": "db.system",
+            "value": "postgresql",
+            "matchType": "strict"
+          }
+        ],
+        "percentage": 100
```
Comment (petergphillips) [inline applicationinsights.dev.json:28]: set to 100% in dev and pre-prod, prod at 5%

File: applicationinsights.dev.json [hunk 2]
```diff
@@ -16,16 +16,49 @@
...
+      },
+      {
+        "telemetryType": "dependency",
+        "attributes": [
+          {
+            "key": "server.address",
+            "value": "(sign-in|prison-api).*",
```
Comment (petergphillips) [inline applicationinsights.dev.json:35]: only requests to prison api are for `GET /api/users/me` and auth are `POST /auth/oauth/token`

File: applicationinsights.dev.json [hunk 3]
```diff
@@ -16,16 +16,49 @@
...
+        ],
+        "percentage": 10
+      },
       {
         "telemetryType": "request",
         "attributes": [
           {
-            "key": "http.url",
-            "value": "https?://[^/]+/health.*",
+            "key": "url.path",
```
Comment (petergphillips) [inline applicationinsights.dev.json:45]: found out can use `url.path` instead which makes this a bit tidier

File: applicationinsights.dev.json [hunk 4]
```diff
@@ -7,7 +7,7 @@
   },
   "instrumentation": {
     "logging": {
-      "level": "DEBUG"
+      "level": "INFO"
```
Comment (petergphillips) [inline applicationinsights.dev.json:10]: I've had a look at `DEBUG` as opposed to `INFO` and nothing really interesting at debug level

**Review by mikehalmamoj — APPROVED** [review]

**Pull request merged**